glib/Cargo.toml

```diff
@@ -59,6 +60,7 @@ log_macros = ["log"]
 dox = ["ffi/dox", "gobject_ffi/dox", "log_macros"]
 compiletests = []
 gio = ["gio_ffi"]
+serialize = ["dep:serde"]
```

Comment:
Why `serialize` instead of `serde`?

Reply:
I wanted to use `serde` but seems like the tests break on it.

https://github.com/gtk-rs/gtk-rs-core/actions/runs/4616156154/jobs/8160815190#step:19:414
https://github.com/gtk-rs/gtk-rs-core/tree/5e2d7622885fd64cfbf7a727273d78368cdc139e

Reply:
Hrm that looks like a bug in `trybuild2`. @GuillaumeGomez you're maintaining that, can you take a look? :)

Reply:
Sure.

Reply:
I published a new version of `trybuild2` with the commits of the upstream `trybuild`.